test/functional/fixtures/regression/gh-711/testcafe-fixtures/test-test.js

```diff
@@ -0,0 +1,37 @@
+import { ClientFunction } from 'testcafe';
```

Comment:
it's a strange filename `test-test.js`
Maybe it's better `index-test.js` or `typing-in-content-editable-test.js`?